projects/usingSoftTimers/src/main.c

```diff
@@ -0,0 +1,51 @@
+
+#pragma once
+
+// Created by Sudhish M on 2020-09-25.
+// usingSoftTimers
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
we generally don't include these headers because they're included in other library headers (sometimes `stdint.h` is used but not the others)